app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  # You should configure your model like this:
```

Comment:
Same. Also, are these 2 controllers actually used in your example? They don't seem to be.

You always must self-review and make sure your PR doesn't include anything that is necessary to your PR.

Reply:
I see there's quite a bit of these controllers, so I'll not leave a comment on each. Please take a look and clean them up.